pkg/executables/kubectl_test.go

```diff
@@ -621,6 +621,135 @@ func TestKubectlGetMachines(t *testing.T) {
 	}
 }
 
+func TestKubectlGetEksaCloudStackMachineConfig(t *testing.T) {
```

Comment:
This might help with these tests :)

https://github.com/aws/eks-anywhere/blob/main/pkg/executables/kubectl_test.go#L1404